css/index.css

```diff
@@ -131,16 +170,58 @@ footer {
     background: black;
 }
 
-footer nav {
-    width: 60%;
+.container footer nav {
+    width: 100%;
     display: flex;
-    justify-content: space-between;
+    justify-content: space-around;
     align-items: center;
     padding: 20px 2%;
     font-size: 14px;
 }
 
```

Comment:
i like that you are thinking about the spacing on these and trying out different justifies